apk/src/com/android/healthconnect/controller/permissions/connectedapps/ConnectedAppsFragment.kt

```diff
@@ -374,11 +374,17 @@ class ConnectedAppsFragment : Hilt_ConnectedAppsFragment() {
     }
 
     private fun setUpEmptyState() {
+        val isPlayStoreAvailable = deviceInfoUtils.isPlayStoreAvailable(requireContext())
+        val isSendAndFeedbackAvailable = deviceInfoUtils.isSendFeedbackAvailable(requireContext())
```

Comment:
I think these lines belong below ` mTopIntro?.title = ...` line. 

Why "And" in `isSendAndFeedbackAvailable` ?